lib/pacemaker/pcmk_sched_native.c

```diff
@@ -1250,7 +1250,7 @@ native_create_actions(pe_resource_t * rsc, pe_working_set_t * data_set)
     gboolean need_stop = FALSE;
     bool need_promote = FALSE;
     gboolean is_moving = FALSE;
-    gboolean allow_migrate = pcmk_is_set(rsc->flags, pe_rsc_allow_migrate)? TRUE : FALSE;
+    gboolean allow_migrate = FALSE;
```

Comment:
I should have commented on this before, but if you turn allow_migrate into a bool, you can get rid of the whole `pcmk_is_set() ? TRUE : FALSE` thing, which looks weird in the first place.

Reply:
Yeah, this file is high on the best-practices list, so I was going to take care of it then